Lecture6/Task1/HttpClientsImplementation/Utils/RequestDataPreparatory.java

```diff
@@ -0,0 +1,29 @@
+package HttpClientsImplementation.Utils;
+
+public class RequestDataPreparatory {
```

Comment:
why it is Preparatory data?

Reply:
In general, DTO objects with only 3 fields can be avoided. It looks like an overhead

Reply:
fixed